src/gsy_e/models/area/__init__.py

```diff
@@ -352,10 +351,6 @@ def cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fal
         if deactivate:
             return
 
-        if self.should_update_child_strategies is True:
```

Comment:
I removed this check because it looks like the sole reason behind its existence is to update child strategies after updating the parent one in the set_grid_fees callback of the redis external market
As I have refactored the mentioned method to use the `area_reconfigure_event` now, there was no need for the `should_update_child_strategies` flag anymore